thirdeye/thirdeye-frontend/app/actions/metrics.js

```diff
@@ -31,87 +42,103 @@ function loading() {
 
 function requestFail() {
   return {
-    type: ActionTypes.REQUEST_FAIL,
+    type: ActionTypes.REQUEST_FAIL
   };
 }
 
 function loadRelatedMetricIds(response) {
```

Comment:
I would say `function  expression` is preferable. But function declaration is fine if we need the hoisting